src/lib/discord/bot.ts

```diff
@@ -1,6 +1,9 @@
 import { Command, ApplicationCommandOptionType } from './api/commands'
+import { Entry, fetchEntries, fetchEntry, countEntries } from '../database/entries'
+import { createSlug } from '../database/index'
 
-// NOTE: do `npm run register-commands` if you change the syntax of any command
```

Comment:
This comment should probably be kept if it's still relevant.

Reply:
This comment is unneeded unless if you add or edit command metadata (such as name, description and choices). Removing in next pushed commit

Reply:
It's only needed when you change command data, such as it's name, desc and choices.

Reply:
Fixed in recent comment, closing convo